site/docs/pages/guides/contribution.mdx

```diff
@@ -65,6 +65,18 @@ onchainkit
 
 ## Workflows
 
+### Storybook
```

Comment:
I think that this section needs more step by step guidance on how to contribute to onchainkit's Storybook. I'm new to Storybook, and this section in its current state does not help me understand how contributing to onchainKit / Storybook works.